src/+otp/+protherorobinson/ProtheroRobinsonProblem.m

```diff
@@ -14,5 +14,13 @@ function onSettingsChanged(obj)
             obj.RHS = otp.RHS(@(t, y) otp.protherorobinson.f(t, y, lambda, phi, dphi), ...
                 'Jacobian', otp.protherorobinson.jacobian(lambda, phi, dphi));
         end
+        
+        function y = internalSolveExactly(obj, t)
+            if ~isequal(obj.Y0, obj.Parameters.Phi(obj.TimeSpan(1)))
+                error('An exact solution is unavailable for this initial condition');
```

Comment:
I would make this a standard error in the OTP namespace, so something like error('OTP:NoExactSolution', 'An exact...

Reply:
All errors should have this I think. I'll save it for another PR, though, and just made issue about it.